components/status/StatusActionsMore.vue

```diff
@@ -32,6 +33,11 @@ const userSettings = useUserSettings()
 const useStarFavoriteIcon = usePreferences('useStarFavoriteIcon')
 
 const isAuthor = $computed(() => status.account.id === currentUser.value?.account.id)
+const statusRoute = $computed(() => {
+  const route = getStatusRoute(status)
+  return withHttps(route.href.slice(1))
+})
+const notLocal = $computed(() => status.url !== statusRoute)
```

Comment:
This is just a helper that will let us know if we should omit additional options in the `more` menu (show original, copy original) if they are redundant

Reply:
also, this version of the `notLocal` test doesn't work on my machine currently – the `status.url` has `http://` whereas the `statusRoute` has `https://`. Not sure if this same issue will happen on staging+production, but it might be better to use the `serverName` approach.

Reply:
Thanks for the heads up @aaga! Switched that out.